src/mo_types/arrange_type.ml

```diff
@@ -92,7 +92,8 @@ module Make (Cfg : Config) = struct
     | Non                    -> Atom "Non"
     | Pre                    -> Atom "Pre"
     | Typ c                  -> "Typ" $$ [con c]
-    | Named (n, t)            -> "Name" $$ [Atom n; typ t]
+    | Named (n, t)           -> "Name" $$ [Atom n; typ t]
+    | Weak t                 -> "weak" $$ [ typ t]
```

Comment:
```suggestion
    | Weak t                 -> "Weak" $$ [ typ t]
```

(not important, just for consistency. this is just debug spew)

Reply:
yup, fixed now